scripts/sct_extract_metric.py

```diff
@@ -46,6 +45,35 @@ def __init__(self):
         self.perlevel = None
 
 
+class _ListLabelsAction(argparse.Action):
+    """This class makes it possible to call the flag '-list-labels' without the need to input the required '-i'."""
+    def __init__(self,
+                 option_strings,
+                 dest=argparse.SUPPRESS,
+                 default=argparse.SUPPRESS,
+                 help=None):
+        super(_ListLabelsAction, self).__init__(
+            option_strings=option_strings,
+            dest=dest,
+            default=default,
+            nargs=0,
+            help=help)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        file_label = os.path.join(namespace.f, param_default.file_info_label)
+        check_file_exist(file_label, 0)
+        default_info_label = open(file_label, 'r')
+        label_references = default_info_label.read()
+        default_info_label.close()
```

Comment:
Small Python style nit, but this can be simplified using `with`:

```suggestion
        with open(file_label, 'r') as default_info_label:
            label_references = default_info_label.read()
```